grpc/cubic.proto

```diff
@@ -184,15 +184,12 @@ message RecognitionConfig {
   // word-level confidence information is returned.  The default is `false`.
   bool enable_word_confidence = 5;
 
-  // This is an optional field.  If this is set to true, the transcripts will be
-  // presented as raw output from the recognizer without any formatting rules
-  // applied.  They will be in all UPPER CASE, numbers and other special
-  // entities would be presented as the spoken words.  If set to `false`,
-  // formatting rules will be applied to all results.  The default is `false`.
-  //
-  // As an example, if the spoken utterance was `here are four words`:
-  //   with this field set to `false`: "Here are 4 words"
-  //   with this field set to 'true' : "HERE ARE FOUR WORDS"
+  // This is an optional field.  If this is set to true, the field
+  // `RecognitionAlternative.raw_transcript` will be populated with the raw
+  // transcripts output from the recognizer will be exposed without any
+  // formatting rules applied.  If this is set to false, that field will be
+  // returned as an empty string.  The RecognitionAlternative.transcript will
+  // always be populated with the formatted text.
```

Comment:
`populated with text formatted according to the server's settings. `

i.e. if the server had no formatting enabled, the value would still be raw.